app/services/journals/create_service.rb

```diff
@@ -572,10 +572,9 @@ def journal_timestamp_sql(notes, attribute)
     end
 
     def timestamp_sql
-      # Use the ruby time instead of the DB's. If those are out of sync,
-      # a journal might otherwise end up having an updated_at before created_at which
-      # will also reflect back to the journaled object.
-      "'#{Time.zone.now.utc.iso8601}'"
+      # Use the timestamp of the statement, not now() or statement_timestamp
+      # as they always return the same value of the start of transaction
+      'statement_timestamp()'
```

Comment:
> the statement_timestamp() function returns the current date and time (including the time zone offset)

Since our date fields are saved without timestamps I feel like this could lead to timezone trouble again if the database's timezone happens to be not UTC. Is there a way to get the statement timestamp without a timezone or rather in a specific (UTC) timezone, perhaps?